### ISSUE TITLE
symfony/security - minor issue in exception message

### ISSUE THREAD

**Issue by alcohol** [symfony/symfony#59328]
### Symfony version(s) affected

6.x and 7.x

### Description

I think the arguments are the wrong way around? The message made sense in 5.4, but since 6.0 it seems to be the wrong way around.

## 5.4
https://github.com/symfony/symfony/blob/2e85f07bced3b6d433b27bd6b9185a096e99ed2e/src/Symfony/Component/Security/Core/Validator/Constraints/UserPasswordValidator.php#L70 and https://github.com/symfony/symfony/blob/2e85f07bced3b6d433b27bd6b9185a096e99ed2e/src/Symfony/Component/Security/Core/Validator/Constraints/UserPasswordValidator.php#L75

## 6.0 and up
https://github.com/symfony/symfony/blob/b101b71ddacfa664485bb09ec6272971e458f49f/src/Symfony/Component/Security/Core/Validator/Constraints/UserPasswordValidator.php#L56

## 7.0 and up
https://github.com/symfony/symfony/blob/df81998b93c9df9031e36f53c20fa162e7639d4b/src/Symfony/Component/Security/Core/Validator/Constraints/UserPasswordValidator.php#L55

### How to reproduce

n/a

**Comment by wouterj** [comment]
Yes, what you're saying looks correct. Are you willing to [open a pull request](symfony.com/doc/current/contributing/code/pull_requests.html) for the 6.4 branch to fix the order?

Status: Reviewed

**Comment by Link1515** [comment]
Hi, @alcohol @wouterj. I'd like to open a PR to fix this issue.

**Comment by alcohol** [comment]
Sorry @wouterj, been quite busy past few days. Submitted this issue as a reminder for myself to open a PR when I have the time. Looks like I don't have to do that anymore, thanks a lot @Link1515 😄
